sources/api/schnauzer/src/helpers.rs

```diff
@@ -244,6 +250,12 @@ mod error {
             target: String,
         },
 
+        #[snafu(display("Failed to parse {} to integer: {}", what, source))]
+        ParseInt {
+            what: String,
+            source: std::num::ParseIntError,
+        },
+
```

Comment:
Same as above. I don't see this being used in schnauzer anywhere.

Reply:
~~Good point - this snafu is not used directly in schnauzer, but is used in `kubernetes.rs`, [L1027](https://github.com/bottlerocket-os/bottlerocket/pull/2219/files#diff-bba9e4e3e46203be2b12f22e0d654ebd270f0b478dd34f40c31d7aa695620f2fR1027) and [L1097](https://github.com/bottlerocket-os/bottlerocket/pull/2219/files#diff-bba9e4e3e46203be2b12f22e0d654ebd270f0b478dd34f40c31d7aa695620f2fR1097).~~

~~Keeping this one.~~

EDIT: It looks like this is actually _not_ useful in schnauzer, but [_is_ useful in `mod.rs`](https://github.com/bottlerocket-os/bottlerocket/pull/2219/files#diff-2907309a5a3d881c962136148bd4340ec869adfa84eae13208d965370543caf4R139).

Removing this definition of the `ParseInt` snafu.